docs/source/traitsui_user_manual/factories_basic.rst

```diff
@@ -395,23 +398,50 @@ See Also
 ::::::::
 ListEditor, TextEditor
 
-DefaultOverride()
-`````````````````
+
+DateEditor()
+````````````
 
 :Suitable for:
-    (any)
+    Date, List(Date) (custom style only)
 :Default for:
-    (none)
+    Date
+:Optional parameters:
+    *allow_future*, *message*, *months*, *multi_select*, *on_mixed_select*,
+    *padding*, *shift_to_select*, *selected_style*, *strftime*, *view*
+
+The DateEditor() displays a Python datetime date object, usually supplied via a
+Date trait.  The simple style shows a date spin control, while the custom style
+shows one (or potentially more in the wx backend) months in a calendar view.
+Dates can be restricted to past-only by setting *allow_future* to False.
+
+The custom style can also be used for the selection of multiple dates as an
+ordered list of dates by setting *multi_select* to True.  The styling of the
+selected dates can be controlled via setting the *selected_style* to a
+CellFormat() instance.
+
+For readonly style, the user can set the message to display if the date value
+is None, and a date format string for use with strftime.
+
+
+DatetimeEditor()
+````````````````
+
+:Suitable for:
+    Datetime
+:Default for:
+    Datetime
+:Optional parameters:
+    *maximum_datetime*, *message*, *minimum_datetime*, *strftime*
 
-The DefaultOverride() is a factory that takes the trait's default editor and
-customizes it with the specified parameters. This is useful when a trait defines
-a default editor using some of its data, e.g. Range or Enum, and you want to
-tweak some of the other parameters without having recreate that data.
+The DatetimeEditor() displays a Python datetime object, usually supplied
+via a Datetime trait.  The simple style shows a datetime spin control, and maximum
+and minimum selectable datetimess can be supplied to restrict the range of values.
```

Comment:
datetimessss, preciousss ...

Reply:
Sorry. Ahem. I don't know what came over me there; let me try again. I believe there may be a typo in this line. Please consider replacing the word "datetimess" with "datetimes".